### PR TITLE
ZCS-830 Universal UI: Styling changes for quick compose dialog from r…

### PR DESCRIPTION
**Pull request by silentsakky**
…eading pane

- quick compose styling changes
- move open in full tab functionality to quick compose part
- removed some css code which I guess would not be needed

### CODE REVIEW

**Comment by silentsakky** [comment]
Yes it was breaking tag bubble which is fixed now

File: WebRoot/js/zimbraMail/mail/view/ZmConvView2.js [hunk 1]
```diff
@@ -1196,6 +1218,7 @@ ZmConvReplyView.prototype._initializeToolbar = function() {
 		var tb = this._replyToolbar = new ZmButtonToolBar(tbParams);
 		tb.addSelectionListener(ZmOperation.SEND, this._convView._sendListener.bind(this._convView));
 		tb.addSelectionListener(ZmOperation.CANCEL, this._convView._cancelListener.bind(this._convView));
+        tb.addSelectionListener(ZmOperation.OPEN_IN_FULL, this._convView._openInFullListener.bind(this._convView));
```
Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/mail/view/ZmConvView2.js]: indentation issue

File: WebRoot/js/zimbraMail/mail/view/ZmConvView2.js [hunk 2]
```diff
@@ -1253,9 +1276,10 @@ function(addresses, type, addrs, used) {
 
 ZmConvReplyView.prototype._resized =
 function() {
-	var bounds = this.boundsForChild(this._input);
+    // Removed sizing for textarea as we are using width: 100%
+    /*var bounds = this.boundsForChild(this._input);
```
Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/mail/view/ZmConvView2.js]: can we remove commented code?

File: WebRoot/js/zimbraMail/mail/view/ZmConvView2.js [hunk 3]
```diff
@@ -1177,12 +1192,19 @@ ZmConvReplyView.prototype._initializeToolbar = function() {
...
 		];
 		var overrides = {};
 		overrides[ZmOperation.CANCEL] = {
             tooltipKey: "cancel",
-            shortcut:   null
+            shortcut:   null,
+            showImageInToolbar: true,
+            showTextInToolbar: true
+        };
+        overrides[ZmOperation.SEND] = {
```
Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/mail/view/ZmConvView2.js]: small, line 1199 and 1205 should be aligned?

Comment (silentsakky) [inline WebRoot/js/zimbraMail/mail/view/ZmConvView2.js]: fixed

File: WebRoot/js/zimbraMail/mail/view/ZmConvView2.js (hunk 1 above)
Comment (silentsakky) [inline WebRoot/js/zimbraMail/mail/view/ZmConvView2.js]: fixed

File: WebRoot/js/zimbraMail/mail/view/ZmConvView2.js (hunk 2 above)
Comment (silentsakky) [inline WebRoot/js/zimbraMail/mail/view/ZmConvView2.js]: fixed

File: WebRoot/templates/mail/Message.template [hunk 4]
```diff
@@ -623,10 +623,10 @@
...
 
 <template id='mail.Message#Conv2Reply'>
-	<div id="${replyToDivId}"></div>
-	<div id="${replyCcDivId}" style="display:none;"></div>
-	 <div>
-	<textarea id="${replyInputId}" class="ReplyTextarea"></textarea>
+	<div id="${replyToDivId}" class="ReplyTo"></div>
+	<div id="${replyCcDivId}" class="ReplyCc" style="display:none;"></div>
+	<div class="ReplyTextareaWrapper">
+		<textarea id="${replyInputId}" class="ReplyTextarea"></textarea>
```
Comment (tarangkhandelwal) [inline WebRoot/templates/mail/Message.template]: extra space here?

Comment (silentsakky) [inline WebRoot/templates/mail/Message.template]: that is to show indentation as textarea is child element of div

**Pull request closed without merging**